server/app.js

```diff
@@ -27,6 +27,11 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*"); // must update
+  next();
+});
+
```

Comment:
i think we can install cors, require it as cors and then do app.use(cors()) for middleware. That's one way I remember doing it.

Reply:
I've done the cors middleware approach before as well

Reply:
Sounds good to me. Did you all want to do that or should I?

Reply:
go ahead and do that unless @DevDelvo decides to..